lib/susedistribution.pm

```diff
@@ -238,6 +238,7 @@ sub init_consoles {
                 password => $testapi::password
             });
         $self->add_console('install-shell', 'tty-console', {tty => 2});
+        $self->add_console('installation',  'tty-console', {tty => check_var('VIDEOMODE', 'text') ? 1 : 7});
```

Comment:
In `installation/logpackages.pm` there's `select_console('installation')` so the test can return to YaST installation. We need to have this console defined for 'svirt' backend.